templates/cassiopeia/scss/blocks/_footer.scss

```
@@ -27,4 +27,14 @@
       }
     }
   }
+
+  @include media-breakpoint-down(md) {
+    .grid-child {
+      flex-direction: column;
+
+      .mod-footer {
+        margin: 0.375rem 0;
```

```suggestion
        margin: .375rem 0;
```
Fix code style error reported by drone here: https://ci.joomla.org/joomla/joomla-cms/40627/1/20

---

I realized this, when I saw CI fail. It's fixed now.

---

Then I was maybe just not patient enough 😉

---

It's alright, you were doing your part😄